bean/internal/driver/seed/004_subscriptions.sql

```diff
@@ -11,12 +11,9 @@ VALUES
 ('10000000-0000-0000-0000-000000000003', '10000000-0000-0000-0000-000000000001', '10000000-0000-0000-0000-000000000001', 'Amazon Prime', 'https://amazon.com', 1500, 1, 'month'),
 ('10000000-0000-0000-0000-000000000004', '10000000-0000-0000-0000-000000000001', '10000000-0000-0000-0000-000000000001', 'Medium', 'https://medium.com', 100, 1, 'month'),
 ('10000000-0000-0000-0000-000000000005', '10000000-0000-0000-0000-000000000001', '10000000-0000-0000-0000-000000000001', 'ChatGPT', 'https://chatgpt.openai.com', 2000, 1, 'month'),
-('10000000-0000-0000-0000-000000000006', '10000000-0000-0000-0000-000000000001', '10000000-0000-0000-0000-000000000002', 'Domain - whatis277.com', 'https://porkbun.com', 1037, 1, 'year'),
-('10000000-0000-0000-0000-000000000007', '10000000-0000-0000-0000-000000000001', '10000000-0000-0000-0000-000000000002', 'Domain - whatistwosevenseven.com', 'https://porkbun.com', 1037, 1, 'year'),
-('10000000-0000-0000-0000-000000000008', '10000000-0000-0000-0000-000000000001', '10000000-0000-0000-0000-000000000002', 'Domain - whatisharvest.com', 'https://porkbun.com', 1037, 1, 'year'),
-('10000000-0000-0000-0000-000000000009', '10000000-0000-0000-0000-000000000001', '10000000-0000-0000-0000-000000000002', 'Domain - whatisbean.com', 'https://porkbun.com', 1037, 1, 'year'),
-('10000000-0000-0000-0000-000000000010', '10000000-0000-0000-0000-000000000001', '10000000-0000-0000-0000-000000000002', 'Domain - whatispear.com', 'https://porkbun.com', 1037, 1, 'year'),
-('10000000-0000-0000-0000-000000000011', '10000000-0000-0000-0000-000000000001', '10000000-0000-0000-0000-000000000002', 'HEY', 'https://hey.com', 34900, 1, 'year')
+('10000000-0000-0000-0000-000000000006', '10000000-0000-0000-0000-000000000001', '10000000-0000-0000-0000-000000000002', 'Domain - whatisbean.com', 'https://porkbun.com', 1037, 1, 'year'),
+('10000000-0000-0000-0000-000000000007', '10000000-0000-0000-0000-000000000001', '10000000-0000-0000-0000-000000000004', 'Insurance', 'https://safetywing.com', 5508, 1, 'month'),
+('10000000-0000-0000-0000-000000000008', '10000000-0000-0000-0000-000000000001', '10000000-0000-0000-0000-000000000004', 'Magazine', '', 099, 6, 'month')
```

Comment:
This is on purpose. 

We need to tests Subs with no providers.

Also with X months, where X > 1